crates/router/src/core/webhooks/incoming.rs

```diff
@@ -725,18 +721,20 @@ async fn payments_incoming_webhook_flow(
             // If event is NOT an UnsupportedEvent, trigger Outgoing Webhook
             if let Some(outgoing_event_type) = event_type {
                 let primary_object_created_at = payments_response.created;
-                Box::pin(super::create_event_and_trigger_outgoing_webhook(
+
+                Box::pin(super::add_bulk_outgoing_webhook_task_to_process_tracker(
                     state,
-                    merchant_context,
-                    business_profile,
+                    &business_profile,
+                    payment_id.get_string_repr(),
                     outgoing_event_type,
-                    enums::EventClass::Payments,
-                    payment_id.get_string_repr().to_owned(),
-                    enums::EventObjectType::PaymentDetails,
-                    api::OutgoingWebhookContent::PaymentDetails(Box::new(payments_response)),
+                    diesel_models::enums::EventClass::Payments,
+                    diesel_models::enums::EventObjectType::PaymentDetails,
```

Comment:
Nit: Can just qualify them as `enums::...`?


```suggestion
                    enums::EventClass::Payments,
                    enums::EventObjectType::PaymentDetails,
```